lib/cli/test/helpers.js

```diff
@@ -0,0 +1,64 @@
+const { sync: spawnSync } = require('cross-spawn');
+const fs = require('fs');
+const path = require('path');
+
+const CLI_PATH = path.join(__dirname, '..', 'bin');
+
+/**
+ * Copy files
+ * @param {String} source - source directory path
+ * @param {String} target - path to the destination directory
+ * @returns {Void}
+ */
+const copyFileSync = (source, target) => {
```

Comment:
As for `rimraf` I think you can find methods doing that in [`fs-extra`](https://github.com/jprichardson/node-fs-extra#methods)

Reply:
Maybe https://github.com/jprichardson/node-fs-extra/blob/master/docs/copy-sync.md?